dvc/parsing/context.py

```diff
@@ -0,0 +1,61 @@
+from collections.abc import Collection, Mapping, Sequence
+
+# for testing purpose
+# FIXME: after implementing of reading of "params".
+TEST_DATA = {
```

Comment:
A bit weird having it here, why not just have it in the tests themselves?

Reply:
For me to quickly test/qa. Nothing to do with actual test. 🙂

Reply:
@skshetry Still don't understand why we need to keep it in the main codebase.

Reply:
Could you please suggest where to keep these? These are not tests, so they are not in "tests" folder, it's only for me to play with. 

It will be most likely be gone in the next PR.

Reply:
@skshetry I guess it is just not clear to why are they there in the first place. If you think you need it for now and you will get rid of it in the next iteration, then please feel free to keep it, I'm fine with it.